core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -505,6 +507,40 @@ public int onStartCommand(Intent intent, int flags, int startId) {
         return Service.START_NOT_STICKY;
     }
 
+    private void skipIntro(Playable playable) {
+        int skipIntro = 0;
+        if (! (playable instanceof FeedMedia)) {
+            return;
+        }
+
+        if (playable.getLastPlayedTime() > 0) {
```

Comment:
Doesn't this mean that the intro is not skipped if you play it for the second time?

Reply:
I have fixed.
1) If you play, autoskip and rewind, it works as you expected.
2) If you play, autoskip, let it end. and come back, it will work as you expect.  The 2nd time, it will skip again